master/reference/typha/configuration.md

```diff
@@ -51,34 +51,13 @@ The Kubernetes API datastore driver reads its configuration from Kubernetes-prov
-   Identity](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#2-spiffe-identity)
-   and provision client certificates with the same identity in their URI SAN
-   field.
-
--  Alternatively you can configure `ClientCN` and provision client certificates
-   with that `ClientCN` value in their Common Name field.
-
-If both those parameters are set, a client certificate only has to match one of
-them; this is intended for when a deployment is migrating from using Common
-Name to using a URI SAN, to express identity.
-
 | Configuration parameter | Environment variable   | Description | Schema |
 | ----------------------- | ---------------------- | ----------- | ------ |
-| `CAFile`                | `TYPHA_CAFILE`         | The full path to the certificate file for the Certificate Authorities that Typha trusts for Felix-Typha communications. | string |
-| `ClientCN`              | `TYPHA_CLIENTCN`       | If set, the Common Name that each connecting client certificate must have. [Default: not set] | string |
-| `ClientURISAN`          | `TYPHA_CLIENTURISAN`   | If set, a URI SAN that each connecting client certificate must have. [Default: not set] | string |
-| `ServerCertFile`        | `TYPHA_SERVERCERTFILE` | The full path to the certificate file for this Typha instance. | string |
-| `ServerKeyFile`         | `TYPHA_SERVERKEYFILE`  | The full path to the private key file for this Typha instance. | string |
-
-{% include {{page.version}}/felix-typha-tls-howto.md %}
+| `CAFile`                | `TYPHA_CAFILE`         | Use this parameter to supply Typha with the path to the file containing the root certificate of the CA that issued the Felix client certificate. Configures Typha to trust the signature on the certificates provided by the Felix client. Example: `/etc/typha/ca.pem` | string |
+| `ClientCN`              | `TYPHA_CLIENTCN`       | If set, the `Common Name` that Felix's certificate must have. If you have enabled TLS on the communications from Felix to Typha, you must set a value here or in `ClientURISAN`. You can set values in both, as well, such as to facilitate a migration from using one to the other. It either matches, the communication succeeds. [Default: none] | string |
+| `ClientURISAN`          | `TYPHA_CLIENTURISAN`   | If set, a URI SAN that Felix's certificate must have. We recommend populating this with a unique [SPIFFE](https://github.com/spiffe/spiffe) ID. If you have enabled TLS on the communications from Felix to Typha, you must set a value here or in `ClientCN`. You can set values in both, as well, such as to facilitate a migration from using one to the other. If either matches, the communication succeeds. [Default: none] | string |
+| `ServerCertFile`        | `TYPHA_SERVERCERTFILE` | Path to the file containing the server certificate issued to Typha. Typha presents this to Felix clients during the TLS handshake. Example: `/etc/typha/cert.pem` | string |
```

Comment:
I think you should add text like "Enables Typha to participate in mutual TLS authentication and identify itself to Felix clients. " here too.

Reply:
Typha is the server in this case so it is not mutual TLS, per se...it is just regular TLS.